templates/boxMenuItem.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import Adapt from 'core/js/adapt';
+import { classes, compile } from 'core/js/reactHelpers';
+
+export default function BoxMenuItem (props) {
+
+  const {
+    displayTitle,
+    _graphic,
+    body,
+    duration,
+    linkText,
+    _isVisited,
+    _isLocked,
+    _isComplete,
+    title,
+    _isOptional,
+    _nthChild,
+    _totalChild
+  } = props;
+
+  const _globals = Adapt.course.get('_globals');
+
+  const durationLabel = duration
+    ? [_globals?._menu?._boxMenu?.durationLabel.trim(), duration].filter(Boolean).join(' ')
+    : '';
+
+  const visited = _isVisited ? _globals?._accessibility?._ariaLabels?.visited : '';
+  const complete = _isComplete ? _globals?._accessibility?._ariaLabels?.complete : '';
+  const completion = complete || visited;
+  const locked = _isLocked ? _globals?._accessibility?._ariaLabels?.locked : linkText;
+  const optional = _isOptional ? _globals?._accessibility?._ariaLabels?.optional : '';
+  const itemCount = compile(_globals?._menu?._boxMenu?.itemCount || '', { _nthChild, _totalChild });
+  const ariaLabel = [
+    completion, locked, `${title}.`, `${itemCount}.`, `${optional}`
+  ].filter(Boolean).join(' ');
+
+  return (
+    <div className="menu-item__inner boxmenu-item__inner">
+
+      {_graphic?.src &&
+      <div className="menu-item__image-container boxmenu-item__image-container">
+        <img className="menu-item__image boxmenu-item__image" src={_graphic.src} aria-hidden="true" />
+      </div>
+      }
+
+      <div className="menu-item__details boxmenu-item__details">
+        <div className="menu-item__details-inner boxmenu-item__details-inner">
+
+          {displayTitle &&
+          <div className="menu-item__title boxmenu-item__title">
+            <div className={classes([
+              'menu-item__title-inner',
+              'boxmenu-item__title-inner',
+              'js-heading'
+            ])}
+            data-a11y-heading-type="menuItem">
+            </div>
+          </div>
+          }
+
+          {_graphic?.alt &&
+          <span className="aria-label" dangerouslySetInnerHTML={{ __html: compile(_graphic.alt) }} />
+          }
+
+          {body &&
+          <div className="menu-item__body boxmenu-item__body">
+            <div className="menu-item__body-inner boxmenu-item__body-inner" dangerouslySetInnerHTML={{ __html: compile(body) }}>
```

Comment:
Perhaps splitting the attributes across multiple lines helps with readability? 
```suggestion
            <div 
              className="menu-item__body-inner boxmenu-item__body-inner" 
              dangerouslySetInnerHTML={{ __html: compile(body) }}
            />
```